keanu-python/keanu/vartypes.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+import pandas as pd
+
+int_types = (int, np.integer)
+
+float_types = (float, np.float)
+
+bool_types = (bool, np.bool_)
+
+primitive_types = int_types + float_types + bool_types
+
+pandas_types = (pd.Series, pd.DataFrame)
+
+const_arg_types = (np.ndarray, ) + pandas_types + primitive_types
```

Comment:
for consistency it's probably worth calling out `numpy_types = (np.ndarray, )` explicitly. I expect in time it will contain more than one item anyway - for example there's a type called `np.mat`.